Changelog.md

```diff
@@ -7,6 +7,17 @@ This project does _not_ adhere to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html) but contrary to it
 every new version is a new major version.
 
+## 88 - 2023-08-15
+
+### Changed
+
+-   `html` package json property has been moved to `nrfConnectForDesktop.html`.
+-   `nrfConnectForDesktop.html` is not opt-in and apps can be built without it.
```

Comment:
This doesn't seem to make sense to me. 

I think either should the “not” be “now” (`nrfConnectForDesktop.html` is now opt-in and apps can be built without it) or the “can” should be “cannot” (`nrfConnectForDesktop.html` is not opt-in and apps cannot be built without it.). I assume the first one is correct.

Reply:
Should be `now`. Thanks for catching that.